### PR TITLE
Feature/hero and nav

### PR DESCRIPTION
**Pull request by kayla-himmel**
- initial hero section that will need to be updated with final designs (I'll add tickets for this and the nav separately)

### CODE REVIEW

File: styles/_containerQueries.scss [hunk 1]
```diff
@@ -0,0 +1 @@
+$cq1024: 1024px;
```
Comment (chpeebles) [inline styles/_containerQueries.scss:1]: i think we forgot to circle back on trying to get this container query variable to work, do we want to remove this for now?

File: styles/sections/_nav.scss [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  }
+}
+
+#hamburger1,
+#hamburger2,
+#hamburger3 {
+  margin-bottom: 6px;
+  display: block;
+  width: calc(100% - 2px);
+  border: 1px solid white;
```
Comment (chpeebles) [inline styles/sections/_nav.scss:54]: for consistency, can we use our `colors.$color-white` variable here?

File: styles/_containerQueries.scss (hunk 1 above)
Comment (kayla-himmel) [reply styles/_containerQueries.scss:1]: Oh good call...I'll delete the file and add a "v2" ticket for that in case we have time for stuff like that.

**Review by chpeebles — APPROVED** [review]
lgtm!

**Pull request merged**